uk.co.compucorp.civicrm.hrcomments/xml/schema/CRM/HRComments/Comment.xml

```diff
@@ -74,4 +76,12 @@
     <add>4.4</add>
   </field>
 
+  <!-- Since these are not real FKs, it's important to have an index for these field for a better performance in SQL JOINs -->
+  <index>
+    <name>index_entity_id</name>
```

Comment:
what about changing the index name to `index_entity_id_entity_name`? By only looking at the name of the index, someone might think it includes only the `entity_id` field